MapboxNavigation/LanesView.swift

```diff
@@ -80,6 +84,30 @@ public class LanesView: UIView {
                 stackView.addArrangedSubview(laneView)
             }
         }
+        
+        if stackView.arrangedSubviews.count > 0 {
+            show()
+        } else {
+            hide()
+        }
+    }
+    
+    public func show(animated: Bool = true) {
+        guard isHidden == true else { return }
+        if animated {
+            UIView.defaultAnimation(0.3, animations: {
+                self.isHidden = false
```

Comment:
This doesn't work for animation. You have to animate the `alpha` channel, not the boolean.

Fun fact: `isHidden` automatically sets to `true` when the `alpha` value has sufficiently approached `0`.

Reply:
Hidden is animatable if the super view is a UIStackView and auto layout is being used. (https://medium.com/@nrewik/easy-animation-with-uistackview-8878b2856ae2)

Reply:
Ah I guess that makes sense.

Reply:
Yeah, TIL!